pkg/admin/prerun/version.go

```diff
@@ -161,12 +221,13 @@ func getVersionOfExecutable() (versionMetadata, error) {
 }
 
 func getVersionOfData() (versionMetadata, error) {
+	klog.InfoS("START reading version file")
```

Comment:
How often are these messages going to show up? How often do we read that file?

Reply:
Once per boot